source_gen/lib/src/library.dart

```diff
@@ -37,10 +37,8 @@ class LibraryReader {
   Iterable<Element> get allElements => [
         element,
         ...element.topLevelElements,
-        // ignore: deprecated_member_use
-        ...element.libraryImports,
-        // ignore: deprecated_member_use
-        ...element.libraryExports,
+        ...element.definingCompilationUnit.libraryImports,
```

Comment:
are the SDK and pkg:analyzer constraints all valid?

Reply:
`analyzer 6.9.0` has `element.definingCompilationUnit.libraryImports`